examples/imagemagick_globus.py

```diff
@@ -28,12 +28,14 @@ def main():
     campaign = Campaign("My ImageMagick Campaign", logger=logger, force_login=False)
 
     # define an activity
+    dir_path = (
```

Comment:
why is dir_path a single-element tuple instead of just a string?

Reply:
Black wrapped it in parentheses to move it down a line and shorten the line length. This is a way to wrap lines in Python. It's not a tuple. It would need to contain a comma to be a tuple. This is a string `("hello")` and this is a tuple `("hello",)`.